data_structures/src/superblock.rs

```diff
@@ -112,7 +128,7 @@ impl SuperBlockState {
     /// Add a vote sent by another peer.
     /// This method assumes that the signatures are valid, they must be checked by the caller.
     pub fn add_vote(&mut self, sbv: &SuperBlockVote) -> AddSuperBlockVote {
-        if self.received_superblocks.contains(sbv) {
+        let r = if self.received_superblocks.contains(sbv) {
```

Comment:
That r is used for show a comment that should be removed so it should be removed